feat: add page for best practices in using system (#366)
openfga/openfga.dev: docs/content/getting-started/overview.mdx

      description: 'Best Practices of Running OpenFGA in Production Environment.',
      to: 'getting-started/running-in-production',
    },

>     {
>       title: 'Data and API Best Practices',
>       description: 'Best Practices of Managing Tuples and Invoking OpenFGA APIs.',
>       to: 'getting-started/tuples-api-best-practices',
>     },

Reviewer: ```suggestion
{
      title: 'Best Practices',
      description: 'Best Practices for Managing Tuples and Invoking OpenFGA APIs.',
      to: 'getting-started/best-practices',
    },
```
